src/bundle/Command/AuditUserDatabaseCommand.php

```diff
@@ -34,13 +35,18 @@ public function __construct(
         UserService $userService,
         Connection $connection
     ) {
-        parent::__construct('ezplatform:user:audit_database');
+        parent::__construct('ibexa:user:audit_database');
```

Comment:
While at it, it's best that we align with conventions:
```suggestion
        parent::__construct('ibexa:user:audit-database');
```